extension/src/test/fixtures/plotsDiff/index.ts

```diff
@@ -467,9 +466,9 @@ export const getMultiSourceOutput = (): PlotsOutput => ({
 export const REVISIONS = [
   EXPERIMENT_WORKSPACE_ID,
   'main',
-  '4fb124a',
-  '42b8736',
-  '1ba7bcd'
+  'exp-e7a67',
+  'test-branch',
+  'exp-83425'
```

Comment:
[F] As I mentioned in the retro I think we need to make the plots test fixture static/duplicate information instead of having all of this transformation logic present. This logic made it really hard to update the integration tests.

Reply:
Although it will lead to more of this:

![image](https://user-images.githubusercontent.com/37993418/232354461-40cf5a17-4057-46a7-9238-05863157cefe.png)